test/lib/pipeline.test.js

```diff
@@ -1837,6 +1868,301 @@ describe('Pipeline Model', () => {
             });
         });
 
+        /* it('throw err - multipleBuildClusterDisabled with cluster annotations - ' +
```

Comment:
can you remove this as well , if not needed ?

Reply:
done